src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/GH1007.MessageMustBeFlattened_Valid.sarif

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.6.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SARIF Functional Testing"
+        }
```

Comment:
I think the _Invalid test has the not flatten message,
the _Valid should have the flatten message.

Reply:
this is output SARIF file generated by test, do you mean the input test SARIF file?